mailer/__main__.py

```diff
@@ -0,0 +1,33 @@
+import time
+
+from .daemon import Daemon
+from .message import UserEmail
+
+
+def main():
+    daemon = Daemon()
+    while True:
+        try:
+            daemon.tick()
+            mail_file = daemon.get_email_file()
+            if not mail_file:
+                time.sleep(5)
+                continue
+
+            email = UserEmail.from_file(mail_file)
+            net = daemon.get_network(email.user)
+            if not net:
+                daemon.error_reporter.report_warning(
+                    "Unable to determine network for user {!r}".format(email.user)
+                )
+                continue
```

Comment:
maybe cleanup/unlink the mail file before the continue, to prevent getting into a loop of picking up and skipping the same dodgy email?

Reply:
In this case, the data may be populated later so trying again is ok

Reply:
However, some logic to avoid trying again in the same second would be good